demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -192,11 +305,12 @@ class ContentList extends React.Component {
 
     const onContentChange = (evt) => {
       const index = +evt.target.value;
-      this.changeContentIndex(index);
+      const { content } = contentsToSelect[index];
+      this.changeContentIndex(index, content);
```

Comment:
Maybe this function is not well-named anymore. Also, do we really need the index now?